packages/react-examples/src/react/TimePicker/TimePicker.Example.tsx

```diff
@@ -15,46 +19,72 @@ const timePickerStyles: Partial<IComboBoxStyles> = {
+
+  const onSelectDate = React.useCallback((selectedDate: Date) => {
+    setDatePickerDate(selectedDate);
+  }, []);
+
+  const onDateTimePickerChange = React.useCallback((time: Date) => {
+    setCurrentTimeString(time.toString());
+  }, []);
+
   const timeRange: ITimeRange = {
     start: 8,
     end: 14,
   };
 
   return (
-    <Stack tokens={stackTokens} styles={stackStyles}>
-      <TimePicker
-        styles={timePickerStyles}
-        useHour12
-        allowFreeform
-        autoComplete="on"
-        label={'TimePicker basic example'}
-        onChange={onChange}
-        defaultValue={new Date('November 25, 2021 09:15:00')}
-        useComboBoxAsMenuWidth
-      />
-      <TimePicker
-        styles={timePickerStyles}
-        showSeconds
-        allowFreeform
-        increments={15}
-        autoComplete="on"
-        label={'TimePicker with non default options'}
-        useComboBoxAsMenuWidth
-        timeRange={timeRange}
-        onChange={onChange}
-      />
-      <TimePicker
-        styles={timePickerStyles}
-        // eslint-disable-next-line react/jsx-no-bind
-        onFormatDate={onFormatDate}
-        useHour12
-        allowFreeform
-        autoComplete="on"
-        label={'TimePicker with custom time string'}
-      />
-    </Stack>
+    <>
+      <Stack tokens={stackTokens} styles={stackStyles}>
+        <TimePicker
+          styles={timePickerStyles}
+          useHour12
+          allowFreeform
+          autoComplete="on"
+          label={'TimePicker basic example'}
+          onTimeChange={onTimeChange}
+          currentDate={new Date('November 25, 2021 09:15:00')}
+          useComboBoxAsMenuWidth
+        />
+        <TimePicker
+          styles={timePickerStyles}
+          showSeconds
+          allowFreeform
+          increments={15}
+          autoComplete="on"
+          label={'TimePicker with non default options'}
+          useComboBoxAsMenuWidth
+          timeRange={timeRange}
+          onTimeChange={onTimeChange}
+        />
+        <TimePicker
+          styles={timePickerStyles}
+          // eslint-disable-next-line react/jsx-no-bind
+          onFormatDate={onFormatDate}
+          useHour12
+          allowFreeform
+          autoComplete="on"
+          label={'TimePicker with custom time string'}
+        />
+      </Stack>
+      <div style={{ paddingTop: '20px', maxWidth: '50%' }}>
+        <Label>{'DatePicker and TimePicker combination'}</Label>
+        <div style={{ display: 'grid', gridTemplateColumns: '1fr 1fr', gridColumnGap: '3px' }}>
+          <DatePicker
+            placeholder="Select a date..."
+            value={datePickerDate}
+            onSelectDate={onSelectDate}
+            minDate={new Date()}
+          />
+          <TimePicker currentDate={datePickerDate} useComboBoxAsMenuWidth onTimeChange={onDateTimePickerChange} />
+        </div>
+        <Text>{`TimePicker selected time: ${currentTimeString}`}</Text>
```

Comment:
This is nice because it shows users the change on the page. It would be great to update your other examples above that use event handlers to do the same.

Reply:
Added time text to all examples.